sdk/storage/AzureStorageBlob/Source/DataStructures/URLSessionTransferManager.swift

```diff
@@ -97,12 +97,6 @@ public final class URLSessionTransferManager: NSObject, TransferManager, URLSess
         operationQueue.maxConcurrentOperationCount = 2
     }
 
-    // Static Singleton
-    public static var shared: URLSessionTransferManager = {
```

Comment:
I have concerns that this will crash. When I tried to make the upload and download pages of the demo app instantiate their own BlobClients (resulting in different transfer managers), I experienced several CoreData crashes.

Reply:
Yep, I haven't tested a lot of this yet, which is why this is still in a draft state. I was working to craft the "ideal" public API. We should discuss how transfer manager lifecycles are handled, how they are associated to client instances, etc. The decisions we make there will have impact on both iOS and Android.